docs/FAQ.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: global
+title: Tachyon FAQ
+---
+
+## Tachyon Frequently Asked Questions.
+
+1. What is Tachyon?
+[Tachyon](http://tachyon-project.org/index.html) is an in-memory distributed, reliable storage system enabling data sharing across cluster at memory speed. Data reliability is achieved by using lineage concept. In memory speed is acheived by caching working file sets, avoiding performing disk reads.
+
+2. What platforms and Java versions does Tachyon run on?
+Currently Tachyon has been tested with Oracle Java 1.7 running on various distributions of Linux.
+
+3. Is there an easy way to see the status and health of a cluster?
+Yes, Once Tachyon is installed, please login into [http://master_ip:19999](http://master_ip:19999)
+
+4. If I add new nodes to the cluster, will tachyon rebalance/move the cached blocks to the newly added nodes in order to balance disk space utilization between the nodes?
+No, Currently re-balancing of data blocks in not available.
+
+5. How do you achieve Tachyon master redundancy?
+We use Tachyon Master/Stand-by architecture. Also, zookeeper can be used to pick a Tachyon master.
+
+6. What is underlayer FS and how many Underlayer FS are supported?
+Underlayer FS is a general distributed filesystems on which tachyon works seamlessly used as long term storage for fault tolerant and to re-compute missing data. Tachyon inherently is not a filesystem.
+
+7. Can I easily add support to other UnderFS?
+Yes, in fact support for other Under FS is in progress by many contributors.
+
+8. In which language is Tachyon developed?
+Tachyon is primarily developed in Java and exposes Java-like File API's for other applications to interact with.
+
+9. What happens if my data set does not fit in memory?
+If the file does not fit in memory, then the file will not be cached, but at the same time all operations are successfully performed.
+
+10. Can I run Tachyon as a standalone for experimenting and validation?
+Yes, Tachyon can be run in a stand-alone mode.Instructions are available [here](http://tachyon-project.org/Running-Tachyon-Locally.html)
+
+11. Can I run Tachyon in a cluster mode?
+Yes, Tachyon can be run in a cluster mode. Instructions are available [here](http://tachyon-project.org/Running-Tachyon-on-a-Cluster.html)
+
+12. Does Tachyon always needs HDFS?
+No, Tachyon runs on different underlayer storage systems like HDFS, Amazon S3, GlusterFS. Instructions on configuration is [available](http://tachyon-project.org/Setup-UFS.html).
+
+13. Does tachyon work along side with other Frameworks?
+Yes, Tachyon works with [hadoop](http://hadoop.apache.org/), [Spark](http://spark.apache.org/), [HBase](http://hbase.apache.org/), [Flink](http://flink.apache.org/), Shark
```

Comment:
Doesn't behaviour for 9 depend on the user requested behaviour?  E.g. `MUST_CACHE` vs `THROUGH`